### PR TITLE
Remove UserPlan class

### PR DESCRIPTION
**Pull request by syedgulahmed**
Removed UserPlan class and User class and their usages.

### CODE REVIEW

File: src/main/java/org/intellij/sdk/codesync/clients/CodeSyncClient.java [hunk 1]
```diff
@@ -73,16 +72,10 @@ public Pair<Boolean, User> getUser(String accessToken) throws RequestError {
...
-                (Long) userPlanObject.get("SIZE"),
-                (Long) userPlanObject.get("FILE_COUNT"),
-                (Long) userPlanObject.get("REPO_COUNT")
-            );
+        try {
             User user = new User(
                 (String) response.get("email"),
-                (Long) response.get("repo_count"),
-                userPlan
+                (Long) response.get("repo_count")
```
Comment (Basit-qc) [inline src/main/java/org/intellij/sdk/codesync/clients/CodeSyncClient.java:78]: We don't need repo_count as well, kindly remove its usage too @syedgulahmed, thanks

**Review by Basit-qc — CHANGES_REQUESTED** [review]
Looks great 👏  suggested 1 change.

**Comment by Basit-qc** [comment]
Kindly also update with master branch, thanks

File: src/main/java/org/intellij/sdk/codesync/clients/CodeSyncClient.java (hunk 1 above)
Comment (syedgulahmed) [reply src/main/java/org/intellij/sdk/codesync/clients/CodeSyncClient.java:78]: @Basit-qc Should we replace User class with string email? User class will have only one field named email.

**Review by Basit-qc — APPROVED** [review]
@syedgulahmed kindly update CHANGELOG with NOT DEPLOYED tag, thanks

**Review by saleem-latif — APPROVED** [review]
Looks Good 👍🏼

**Review by saleem-latif — APPROVED** [review]
Looks Good 👍🏼

**Pull request merged**